gulpfile.js

```diff
@@ -69,7 +91,8 @@ const javascripts = {
       ]
     }).then(bundle => {
       return bundle.write({
-        file: paths.dist + 'javascripts/govuk-frontend-details.js',
+        dir: paths.dist + 'javascripts/',
+        entryFileNames: '[name]-[hash].js',
```

Comment:
It took me a long time to find the code that generates `[hash]`, to be sure it will always be 8 chars long. Would have been helpful to have a reference in the commit message that introduces this: https://github.com/rollup/rollup/search?q=digest